app/_gateway_entities/route.md

```diff
@@ -4,57 +4,171 @@ content_type: reference
 
-You can also configure routes with:
+Routes can also define rules that match requests to associated Services. Because of this, one Route can reference multiple endpoints. Once a Route is matched, {{site.base_gateway}} proxies the request to its associated Service. A basic Route should have a name, path or paths, and reference an existing Service.
 
-* Protocols: The protocol used to communicate with the upstream application.
-* Hosts: Lists of domains that match a route
-* Methods: HTTP methods that match a route
-* Headers: Lists of values that are expected in the header of a request
-* Redirect status codes: HTTPS status codes
-* Tags: Optional set of strings to group routes with
+When you configure Routes, you can also specify the following:
 
-## Route and service interaction
+* **Protocols:** The protocol used to communicate with the [upstream application](/gateway/entities/upstream/).
+* **Hosts:** Lists of domains that match a Route
+* **Methods:** HTTP methods that match a Route
+* **Headers:** Lists of values that are expected in the header of a request
+* **Redirect status codes:** HTTPS status codes
+* **Tags:** Optional set of strings to group Routes with
 
-Routes, in conjunction with [services](/gateway/entities/service/), let you expose your services to applications with {{site.base_gateway}}. {{site.base_gateway}} abstracts the service from the applications by using routes. Since the application always uses the route to make a request, changes to the services, like versioning, don’t impact how applications make the request. Routes also allow the same service to be used by multiple applications and apply different policies based on the route used.
+## Route and Service interaction
 
-For example, if you have an external application and an internal application that need to access the `example_service` service, but the external application should be limited in how often it can query the service to assure no denial of service. If a rate limit policy is configured for the service when the internal application calls the service, the internal application is limited as well. Routes can solve this problem.
+Routes, in conjunction with [Services](/gateway/entities/service/), let you expose your Services to applications with {{site.base_gateway}}. {{site.base_gateway}} abstracts the Service from the applications by using Routes. Since the application always uses the Route to make a request, changes to the Services, like versioning, don’t impact how applications make the request. Routes also allow the same Service to be used by multiple applications and apply different policies based on the Route used.
```

Comment:
```suggestion
Routes, in conjunction with [Services](/gateway/entities/service/), let you expose your Services to applications with {{site.base_gateway}}. {{site.base_gateway}} abstracts the Service from the applications by using Routes. Since the application always uses the Route to make a request, changes to the Services, like versioning, don’t impact how applications make the request.
```
Moving this sentence to intro the example instead, since the example is about that.